### ISSUE TITLE
Double-blind reviews

### ISSUE THREAD

**Issue by effigies** [OpenNeuroOrg/openneuro#2824]
### What would you like to see added?

The current review links allow anonymous access to the dataset, but do not blind the dataset authors' identities from the reviewer. If the dataset is provided to a reviewer as part of a double-blinded peer review, this breaks the blind.

Authors can choose not to identify themselves, but if they haven't considered the possibility before making their first snapshot, the damage is done.

### Alternatives

Update docs and link to them during the snapshot process.

### Do you have any interest in helping implement the feature?

Yes, but I would need guidance

**Comment by nellh** [comment]
This would require hiding the uploader/author information when viewed from a review link? I'm not sure how that would be accomplished without limiting access to the dataset itself since it leaks via commit information and data within the dataset.

**Comment by effigies** [comment]
Yeah, possibly only allowing access to the most recent snapshot, and not granting `git clone` access to reviewer links? Then we could put an advisory next to "Create reviewer link"?

**Comment by effigies** [comment]
From today's meeting, possibly the cleanest approach would be to run an anonymization process on the latest tag, create an orphan branch to eliminate any git history, and push as a new dataset that the review link has access to.

We probably want to create a separate namespace, such as `datasets/rv00WXYZ`.

I think ideally we would keep the existing setup for non-blinded datasets, but it might be easier to have one code path for all reviews. If that's the case, we could make the anonymization script an optional step.

Once a dataset is published, review datasets should redirect to the full dataset.
